Quaver.Shared/Screens/Gameplay/UI/JudgementHitBurst.cs

```diff
@@ -86,6 +87,9 @@ public void ChangeJudgementFrames(Judgement j)
         /// <param name="j"></param>
         public void PerformJudgementAnimation(Judgement j)
         {
+            // If show hit accuracy is disabled, do nothing
+            if (!ConfigManager.ShowHitAccuracy.Value) return;
```

Comment:
If you're hiding only marvelous and perfects, I would expect there to be some sort of condition here that has that. This currently hides all of them.